Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV["TERM"] = "xterm-256color"
+ENV["GOPATH"] = "/go"
+ENV["LC_ALL"] = "en_US.UTF-8"
+
+Vagrant.configure("2") do |config|
+  # config.vm.box = "fedora/25-cloud-base"
+  # config.vm.provision "shell", inline: "dnf install -y go git docker"
+  config.vm.box = "ubuntu/zesty64"
+  config.vm.provision "shell", inline: "DEBIAN_FRONTEND=noninteractive apt-get install -y golang git docker.io systemd-container tmux"
```

Comment:
I didn't know there was a non-docker build. But that line with `usermod` works for me ;)